packages/react/src/primitives/shared/index.ts

```diff
@@ -1 +1,2 @@
 export { ComponentClassNames, ComponentClassObject } from './constants';
+export { classNameModifier, classNameModifierByFlag } from './utils';
```

Comment:
Should these utils be moved to the ui package?

Reply:
yea probably